platforms/hyperledger-indy/configuration/roles/create/helm_component/pool_genesis/README.md

```diff
@@ -0,0 +1,49 @@
+## create/helm_component/pool_genesis
+This role create the config map value file for storing domain genesis for Indy cluster.
+
+## Tasks:
+### 1. Ensures {{ release_dir }}/{{ component_type }}/{{ component_name }} dir exists
+This task ensure, that release folder for value file exists.
+It the folder doesn't exist, then creates them.
+
+#### Variables:
+ - release_dir: Release directory, where are stored generated files for gitops. Default value: {{ playbook_dir }}/../../../{{ gitops.release_dir }}
+ - gitops: A object, which contains data of organization's gitops from network.yaml file.
+ - component_name: Name of component.
+
+### 2. Generate pool genesis for organization
+This task generate pool genesis with data from crypto, which is in Vault.
+This task need baf-ac token for getting public data from Vault.
+Result is a jsons for each organizations' stewards
```

Comment:
I would use some more complex description of the pool genesis
https://github.com/hyperledger/indy-node/blob/master/docs/source/transactions.md#genesis-transactions